### DISCUSSION TITLE
simplewing sample and wake calculation

### DISCUSSION THREAD

**Discussion by pablodecastillo** [byuflowlab/FLOWUnsteady#117] [Q&A]
Hello Eduardo,
I was debugging the wing sample, and when you calculate the system, you compute the semi-infinite vortices to take account the kutta condition, and the particles from Vpm too, my question is if your semi-infinite vortices has a lot influence over the particles movement. 
Thanks

**Comment by EdoAlvarezR** [comment]
Hi Pablo! Could you please point out in the code where that happens? I'm 99% sure that the semi-infinite vortices are never used in FLOWUnsteady (except for the quasi-steady solver).
